aten/src/ATen/native/mkldnn/xpu/detail/Utils.h

```diff
@@ -94,6 +96,6 @@ dnnl::memory dnnl_memory_from_host_scalar(
   dnnl::memory::desc md = get_onednn_md(holder);
   dnnl::memory mem = make_onednn_memory(md, engine, holder.data_ptr());
   return mem;
-}
+};
```

Comment:
@Skylion007 Don't we have no-extra-semi linter now? 
```suggestion
}
```